settings.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+  ~ Copyright 2020 Google LLC.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                          https://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <mirrors>
+    <!-- Google's Maven Central mirror for Americas
+     https://maven-central.storage.googleapis.com/index.html
+    -->
+    <mirror>
+      <id>google-maven-central</id>
+      <name>GCS Maven Central mirror</name>
+      <url>https://maven-central.storage-download.googleapis.com/maven2/</url>
```

Comment:
"maven2/" is correct one.

Reply:
Is there anyway we can set one repo as a backup for another?

Reply:
Nice idea. Updated.

Reply:
@elharo It didn't work. I tried the below and the second mirror (real Maven Central) was unused:

```
suztomo-macbookpro44% cat ~/.m2/settings.xml
<!--
  ~ Copyright 2020 Google LLC.
  ~
  ~ Licensed under the Apache License, Version 2.0 (the "License");
  ~ you may not use this file except in compliance with the License.
  ~ You may obtain a copy of the License at
  ~
  ~     http://www.apache.org/licenses/LICENSE-2.0
  ~
  ~ Unless required by applicable law or agreed to in writing, software
  ~ distributed under the License is distributed on an "AS IS" BASIS,
  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  -->

<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
                          https://maven.apache.org/xsd/settings-1.0.0.xsd">
  <mirrors>
    <mirror>
      <!-- Google's Maven Central mirror for Americas
       https://maven-central.storage.googleapis.com/index.html
      -->
      <id>google-maven-central</id>
      <name>GCS Maven Central mirror</name> 
      <url>https://maven-central.storage-download.googleapis.com/repos/central/data/</url> <!-- this is outdated URL -->
      <mirrorOf>central</mirrorOf>
    </mirror>
    <mirror>
      <id>maven-central</id>
      <name>Maven Central</name>
      <url>https://repo1.maven.org/maven2</url>
      <mirrorOf>central</mirrorOf>
    </mirror>
  </mirrors>
</settings>
suztomo-macbookpro44% mvn clean install
[INFO] Scanning for projects...
...
[ERROR]     Non-resolvable import POM: Failure to find com.google.cloud:google-cloud-bom:pom:0.122.5-alpha in https://maven-central.storage-download.googleapis.com/repos/central/data/ was cached in the local repository, resolution will not be reattempted until the update interval of google-maven-central has elapsed or updates are forced @ com.google.cloud:libraries-bom:4.4.0-SNAPSHOT, /Users/suztomo/cloud-opensource-java/boms/cloud-oss-bom/pom.xml, line 196, column 19 -> [Help 2]
[ERROR]     'dependencies.dependency.version' for com.google.cloud:google-cloud-core:jar is missing. @ line 57, column 17
[ERROR]     'dependencies.dependency.version' for com.google.cloud:google-cloud-storage:jar is missing. @ line 61, column 17
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
```